src/components/biggive-progress-bar/biggive-progress-bar.tsx

```diff
@@ -24,7 +24,7 @@ export class BiggiveProgressBar {
     return (
       <div class={'progress-bar progress-bar-' + this.colourScheme + ' space-below-' + this.spaceBelow}>
         <div class="slider">
-          <div class="progress" style={{ width: `${this.counter}%` }}></div>
+          <div class="progress" style={{ width: `${this.counter >= 100 ? 100 : this.counter}%` }}></div>
```

Comment:
Could simplify this slightly to 
```suggestion
          <div class="progress" style={{ width: `${Math.min(this.counter, 100)}%` }}></div>
```

It would be nice if more languages had a clamp function like [css](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp) , I think that's a bit more intuitive than max and min.

Why is it possible for the counter to be more than 100?

Reply:
@bdsl Thanks for the suggestion, committed. To answer your question: the counter can be more than 100% when campaigns exceed their targets.